code/sorting/bubble_sort/README.md

```diff
@@ -37,6 +37,11 @@ Now, the array is already sorted, but our algorithm does not know if it is compl
 ## Further reading
 [Wikipedia - Bubble sort](https://en.wikipedia.org/wiki/Bubble_sort)
 
+Best video tutorial
```

Comment:
Best is a bit of a strong word and an opinion; Change it to something like good.